bin/main.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../lib/logic.rb'
+
+game = Game.new
+puts 'A Tic-Tac-Toe game'
```

Comment:
User interaction is fine here. You could however benefit from moving all of it to a `displayController.rb` file and calling them from here, thereby isolating your user interaction from game logic completely.